packages/ui-codemods/README.md

```diff
@@ -8,22 +8,26 @@ The ui-codemods should make it easier to deal with API changes when upgrading In
 
 [![npm][npm]][npm-url]&nbsp;
 [![build-status][build-status]][build-status-url]&nbsp;
-[![MIT License][license-badge]][LICENSE]&nbsp;
+[![MIT License][license-badge]][license]&nbsp;
 [![Code of Conduct][coc-badge]][coc]
 
 ### Installation
 
 The codemod scripts can be installed via the following command:
+
 ```sh
 yarn add @instructure/ui-codemods
 ```
 
 The configuration files are located in the [instui-config](#instui-config) package.
 This can be installed via the following command:
+
 ```sh
 yarn add @instructure/instui-config
 ```
 
+### If you are in the process of updating from InstUI v7 to v8, run these codemods from the latest version of this package!
```

Comment:
This is just nitpicking, but I would put this into a `<ToggleBlockquote>` or Alert or sth, because this (###) makes it render as a header, which is not ideal neither visually nor semantically.